drivers/wifi/nrf700x/osal/fw_if/umac_if/inc/fw/host_rpu_sys_if.h

```diff
@@ -862,6 +864,11 @@ struct nrf_wifi_cmd_sys_init {
 	 *  alive frame.
 	 */
 	unsigned int keep_alive_period;
+	/** The RPU uses this value (in seconds) to determine the time interval between
+	 *  a null frame sent after a TIM indication beacon with the PWR MGT flag set to
+	 *  0 and a null frame with the PWR MGT flag set to 1.
+	 */
+	unsigned int qos_null_frame_interval;
```

Comment:
Also have a `ps` in the name somewhere here?

Reply:
The naming is followed as per latest main. It can be changed later if required.